akka-actor/src/main/scala/akka/japi/JavaAPI.scala

```diff
@@ -60,7 +60,7 @@ trait Predicate[T] {
  * Additional tuple types for 3 to 22 values are defined in the `akka.japi.tuple` package, e.g. [[akka.japi.tuple.Tuple3]].
  */
 @SerialVersionUID(1L)
-case class Pair[A, B](first: A, second: B) {
+case class Pair[+A, +B](first: A, second: B) {
```

Comment:
I'm not sure about this... It feels like a pretty big change, that could affect a lot of things. On the other hand, I think it is right? I mean, Scala Tuples are covariant, and Pair is immutable so it should be OK...
This also brings up the question of what to do with all the other japi.TupleN types...